### PR TITLE
Improvements to doc on control plane installation

### PR DESCRIPTION
**Pull request by jijiechen**
Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits)
- Yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?
- Yes

This fixes https://github.com/kumahq/kuma-website/issues/1509

### CODE REVIEW

File: app/_src/production/deployment/multi-zone.md [hunk 1]
```diff
@@ -108,7 +108,8 @@ Data plane proxy changes will be propagated within their zones.
 
 ### Zone control plane offline
 
-- New data plane proxies won't be able to join the mesh.
+- New data plane proxies won't be able to join the mesh. This includes new instances (Pod/VM) that are newly created by automatic deployment mechanisms (e.g. a rolling update process), this means a control plane connection failure could block "update by replacement processes" of applications and events that involves new instances to be created.
```
Comment (lahabana) [inline app/_src/production/deployment/multi-zone.md:111]:
```suggestion
- New data plane proxies won't be able to join the mesh. This includes new instances (Pod/VM) that are newly created by automatic deployment mechanisms (e.g. a rolling update process), this means a control plane connection failure could block "rolling updates" of applications and events that involves new instances to be created.
```

File: app/_src/production/deployment/multi-zone.md [hunk 2]
```diff
@@ -108,7 +108,8 @@ Data plane proxy changes will be propagated within their zones.
...
+- On mTLS enabled meshes, a data plane proxy may fail to refresh its client certificate prior to expiry (defaults to 24h, could be configured at mesh level), thus traffic from/to this data planes will be blocked.
```
Comment (lahabana) [inline app/_src/production/deployment/multi-zone.md:112]:
```suggestion
- On mTLS enabled meshes, a data plane proxy may fail to refresh its client certificate prior to expiry (defaults to 24 hours), thus causing traffic from/to this data planes to fail.
```

File: jekyll.yml [hunk 3]
```diff
@@ -77,6 +77,7 @@ mesh_product_name: Kuma
 mesh_product_name_path: kuma
 mesh_namespace: kuma-system
 mesh_cp_name: kuma-control-plane
+mesh_cp_zone_sync_name_prefix: kuma-
```
Comment (lahabana) [inline jekyll.yml:80]: You should add it the `jekyll-dev.yml` too

Comment (jijiechen) [reply jekyll.yml:80]: Yes, already added.

File: app/_src/production/deployment/multi-zone.md (hunk 2 above)
Comment (jijiechen) [reply app/_src/production/deployment/multi-zone.md:112]: Thanks, I will also fix other files in a new commit.

**Comment by jijiechen** [comment]
Please hold on, I'll rebase from the new master with 2.5.x docs.

**Comment by jijiechen** [comment]
> Please hold on, I'll rebase from the new master with 2.5.x docs.

This is done now.

File: app/_src/production/cp-deployment/multi-zone.md [hunk 4]
```diff
@@ -302,6 +303,19 @@ For production use a certificate signed by a trusted CA. See [Secure access acro
 
 ### Verify control plane connectivity
 
+If your global control plane is installed on Kubernetes, you'll need to configure your `kumactl` like so:
```
Comment (lahabana) [inline app/_src/production/cp-deployment/multi-zone.md:306]:
```suggestion
If your global control plane runs on Kubernetes, you'll need to configure your `kumactl` like so:
```

File: app/_src/production/cp-deployment/stand-alone.md [hunk 5]
```diff
@@ -40,9 +40,9 @@ Once {{site.mesh_product_name}} is up and running, data plane proxies can now {%
 When the mode is not specified, {{site.mesh_product_name}} will always start in `standalone` mode by default.
 {% endtip %}
 
-#### Optional: Docker authentication
+#### Optional: Control plane authentication
```
Comment (lahabana) [inline app/_src/production/cp-deployment/stand-alone.md:43]:
```suggestion
#### Optional: control plane authentication
```

File: app/docs/1.8.x/deployments/multi-zone.md [hunk 6]
```diff
@@ -428,7 +428,8 @@ Data plane proxy changes will be propagated within their zones.
 
 ### Zone control plane offline
 
-* New data plane proxies won't be able to join the mesh.
+* New data plane proxies won't be able to join the mesh. This includes new instances (Pod/VM) that are newly created by automatic deployment mechanisms (for example, a rolling update process), meaning a control plane connection failure could block updates of applications and events that involves new instances to be created.
```
Comment (lahabana) [inline app/docs/1.8.x/deployments/multi-zone.md:431]:
```suggestion
* New data plane proxies won't be able to join the mesh. This includes new instances (Pod/VM) that are newly created by automatic deployment mechanisms (for example, a rolling update process), meaning a control plane connection failure could block updates of applications and events that create new instances.
```

File: app/docs/1.8.x/deployments/stand-alone.md [hunk 7]
```diff
@@ -65,7 +65,8 @@ When the mode is not specified, Kuma will always start in `standalone` mode by d
 
 #### Control plane offline
 
-* New data plane proxies won't be able to join the mesh.
+* New data plane proxies won't be able to join the mesh. This includes new instances (Pod/VM) that are newly created by automatic deployment mechanisms (for example, a rolling update process), meaning a control plane connection failure could block updates of applications and events that involves new instances to be created.
```
Comment (lahabana) [inline app/docs/1.8.x/deployments/stand-alone.md:68]:
```suggestion
* New data plane proxies won't be able to join the mesh. This includes new instances (Pod/VM) that are newly created by automatic deployment mechanisms (for example, a rolling update process), meaning a control plane connection failure could block updates of applications and events that create new instances.
```

**Review by lahabana — APPROVED** [review]
@jijiechen BTW in the future no need to update docs for non "single version" docs. These are considered deprecated versions and we should update the docs.

File: app/_src/production/deployment/multi-zone.md [hunk 8]
```diff
@@ -108,7 +108,8 @@ Data plane proxy changes will be propagated within their zones.
 
 ### Zone control plane offline
 
-- New data plane proxies won't be able to join the mesh.
+- New data plane proxies won't be able to join the mesh. This includes new instances (Pod/VM) that are newly created by automatic deployment mechanisms (for example, a rolling update process), meaning a control plane connection failure could block updates of applications and events that involves new instances to be created.
```
Comment (lahabana) [inline app/_src/production/deployment/multi-zone.md:111]:
```suggestion
- New data plane proxies won't be able to join the mesh. This includes new instances (Pod/VM) that are newly created by automatic deployment mechanisms (for example, a rolling update process), meaning a control plane connection failure could block updates of applications and events that create new instances.
```

**Pull request merged**